.circleci/config.yml

```diff
@@ -1,37 +1,20 @@
-version: 2
+version: 2.1    
 jobs:
   build:
     docker:
-      - image: ux3d/gltf-blender-io
+      - image: p3din/gltf-blender-io
+    parallelism: 4
     steps:
       - checkout
-      - run:
-          name: Install Blender Plugin (2.79)
-          command: rm -r /opt/blender279b/2.79/scripts/addons/io_scene_gltf2 && cp -r addons/io_scene_gltf2 /opt/blender279b/2.79/scripts/addons/io_scene_gltf2
-      - run:
-          name: Install Blender Plugin (2.80)
-          command: rm -r /opt/blender28/2.80/scripts/addons/io_scene_gltf2 && cp -r addons/io_scene_gltf2 /opt/blender28/2.80/scripts/addons/io_scene_gltf2
-      - run:
-          name: Prepare Tests
-          command: cd tests && npm install
       - run:
           name: Run Tests
-          command: cd tests && npm test
-      - run:
-          name: Store Results
-          when: always
-          command: cp -r tests/mochawesome-report /tmp/report
-      - run:
-          name: Store Scenes
-          when: always
-          command: cp -r tests/scenes /tmp/scenes
-      - run:
-          name: Store glTFs
-          when: always
-          command: cp -r tests/roundtrip /tmp/roundtrip
-      - store_artifacts:
-          path: /tmp/report
-      - store_artifacts:
-          path: /tmp/scenes
+          command: |
```

Comment:
Was there a specific reason to get rid of the separation of steps? Maybe we don't need as many, but I think it makes sense to separate pre and post steps from the actual test execution.

Reply:
Must admit I was a bit unsure about how steps were handled for parallel builds, so tried to put everything in one. I agree having the pre-setup in separate step makes sense, will fix